src/plugins/kv-analytics-plugin.js

```diff
@@ -128,7 +137,10 @@ export default Vue => {
 	Vue.prototype.$fireServerPageView = () => {
 		const to = { path: window.location.pathname };
 		const from = { path: document.referrer };
-		kvActions.pageview(to, from);
+		// delay pageview call to ensure window.performance.timing is fully populated
+		setTimeout(() => {
+			kvActions.pageview(to, from);
+		}, 500);
```

Comment:
I feel like this should be some sort of dom-interactive check if we want to ensure that it's populated